dist/README.md

```diff
@@ -0,0 +1,27 @@
+# Installing
+
+Pick the binary that matches your operating system and architecture. For
+example, on my Macbook Pro I would use `ftree-darwin-amd64`. Copy that somewhere
+in your path and rename to `ftree`. Presto bango.
+
+## Verifying
+
+The best way to verify the release is to examine the GPG signature. You can find
+my public key at http://calebthompson.io/pubkey.asc,
+<https://pgp.mit.edu/pks/lookup?op=get&search=0x1621ADC2A0ACE70A>, and via GnuPG
+with `gpg --recv-keys A0ACE70A`.
```

Comment:
How is this more secure than verifying with the sha? Presumably if I access the git repo to edit the sha, I can also edit the git repo to edit this URL or key id?